src/ui/components/Rating/index.js

```diff
@@ -50,16 +55,19 @@ export class RatingBase extends React.Component {
       const props = {
         className: classNames('Rating-choice', {
           'Rating-selected-star': rating && thisRating <= rating,
+          // Half stars are the result of averages rounded to the nearest
+          // 0.5 place; only show them when readOnly is true.
+          'Rating-half-star': (readOnly && rating && thisRating > rating &&
```

Comment:
We don't really need to round it anymore because if rating was `3.6` and we're on `rating === 4` then `thisRating (4) - 0.5 <= rating (3.6)` is true. So we don't have to round anywhere here.